geoips/plugins/modules/readers/ahi_hsd.py

```diff
@@ -995,8 +1005,9 @@ def call(
     elif self_register:
         if self_register not in DATASET_INFO:
             raise ValueError(
-                "Unrecognized resolution name requested for self registration: {}".format(
-                    self_register
+                """Unrecognized resolution name requested for self registration:
```

Comment:
Probably something like
"line1" 
f"{self_register}"
"line2"

Reply:
fixed this